src/server/plugins/engine/plugin.ts

```diff
@@ -80,9 +80,12 @@ export const plugin = {
     const forms = server.app.forms
 
     configs.forEach((config) => {
+      const basePath =
+        formBasePath === '' ? config.id : `${formBasePath}/${config.id}`
```

Comment:
Another suggestion, fewer edits to make in your next PR

```suggestion
      const basePath = formBasePath ? `${formBasePath}/${config.id}` : config.id
```